src/js/user-directory/components/steps/StepTabs.js

```
@@ -0,0 +1,526 @@
+        const orderedTabs = [];
+        if (savedOrder.length > 0) {
+            savedOrder.forEach(key => {
+                if (tabsMap[key]) {
+                    orderedTabs.push(tabsMap[key]);
+                    delete tabsMap[key];
+                }
+            });
+        }
+        
+        // Add remaining tabs
+        Object.values(tabsMap).forEach(tab => orderedTabs.push(tab));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)